Add 11.41 release notes (#567)
mondoohq/docs: releases/2025-02-11-mondoo-11.41-is-out.md

---
slug: mondoo-11.41-is-out/
title: Mondoo 11.41 is out!
description: Announcing the 11.41 release of Mondoo with RDS cluster support, improved asset navigation, new resources, and more
authors: [tim]
image: /img/releases/2025-02-11-mondoo-11.41-is-out/rds.png
tags: [release, mondoo]
---

## 🥳 Mondoo 11.41 is out! This release includes RDS cluster support, improved asset navigation, new resources, and more!

Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)

---

## 🧹 IMPROVEMENTS

### AWS RDS Cluster support

> AWS account scans now include automatic discover of RDS clusters. Cluster assets will appear automatically in the Assets page in a new RDS Clusters group and each asset includes full inventory information.

Reviewer: ```suggestion
AWS account scans now include automatic discovery of RDS clusters. Cluster assets appear on the Assets page in a new RDS Clusters group and each asset includes full inventory information.
```

![RDS Cluster Scan](/img/releases/2025-02-11-mondoo-11.41-is-out/rds.png)

### More view options for asset lists

> We've updated the Assets page to make finding and selecting the assets you care about easier than ever. Improved search results include just the asset groups with results and a new empty state makes it more clear when no assets matched your search. New group and list icons let you switch between assets in platform groups or a single list making it possible to sort and select all assets of a certain asset group at once.

Reviewer: ```suggestion
We've updated the Assets page to make finding and selecting the assets you care about easier than ever. Improved search results show just the asset groups that include results. A new empty state makes it more clear when no assets match your search. New group and list icons let you switch between views: organize by platform groups or display a single list so you can sort and select all assets of a certain asset group at once.
```

![Asset selection](/img/releases/2025-02-11-mondoo-11.41-is-out/assets.gif)

### Resource updates

#### azure.subscription.cloudDefenderService

- New `settingsMCAS`, `settingsWDATP`, and `settingsSentinel` fields using the new `azure.subscription.cloudDefenderService.settings` resource

#### azure.subscription.cloudDefenderService.securityContact

> - Deprecated `alertNotifications` in favor of the new `notificationSources` field

Reviewer: ```suggestion
- Deprecate `alertNotifications` in favor of the new `notificationSources` field
```

#### azure.subscription.webService.appsite

- New `ftp` and `scm` fields using the new `azure.subscription.webService.appsite.basicPublishingCredentialsPolicies` resource

#### microsoft

- New `users` field using the new `microsoft.users` resource
- New `roles` field using the new `microsoft.roles` resource

#### sshd.config

- New `context` field using the new `file.context` resource

## 🐛 BUG FIXES AND UPDATES

- Prefer the `--token` command line flag over the `GITHUB_TOKEN` environmental variable when both are present.
- Display asset platforms in search results.
- Preselect ticket email recipient values when only a single option exists in the drop down menu.

> - Don't show the "All Assets" and "Selected Assets" options for ticket creation when only a single asset exists.

Reviewer: ```suggestion
- When a user selects "Create ticket," don't show the "All Assets" and "Selected Assets" options if there's only one asset.
```

- Don't show policies or checks counts on third-party integration pages.
- Show a friendly empty state when a filter on the Assets page returns no results.
- Don't show empty asset groups when filtering on the Assets page.
- Display a friendly error if an exception cannot be created.

> - Don't require new users in a private instance to active the region.

Reviewer: ```suggestion
- Don't require new users in a private instance to activate the region.
```

> - Improve the reliability of AWS region fetching in low privilege environments.

Reviewer: ```suggestion
- Improve the reliability of AWS region fetching in low-privilege environments.
```

- Fix risk factors settings in the security model configuration not applying in some cases.
- Adjust score weighting for risk factors to better prioritize findings.

> - Don't zip single file PDF reports so they can be opened directly in the browser.

Reviewer: ```suggestion
- Don't zip one-file PDF reports so users can open them directly in the browser.
```
